pkg/gardenlet/apis/config/types.go

```diff
@@ -403,6 +406,52 @@ type SNIIngress struct {
 	Labels map[string]string
 }
 
+// ETCDConfig contains ETCD related configs
+type ETCDConfig struct {
+	// ETCDController contains config specific to ETCD controller
+	// +optional
```

Comment:
Do not add `+optional` tags for internal APIs

Reply:
You suggested earlier to add `optional` tag. Right? I already removed them

Reply:
Ref https://github.com/gardener/gardener/pull/5037#discussion_r758158307